src/app/(ig)/IgDetail.module.css

```
@@ -12,288 +11,84 @@
 }
 
-.SigActionRow {
-  display: flex;
-  gap: 0.5rem;
-  flex-wrap: wrap;
-  margin: 1rem 0 0.75rem;
+.content {
+  line-height: 1.75;
+  font-size: 1rem;
+  color: var(--color-text-subtle);
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)